src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/attributes/edit.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+    import { goto, invalidate } from '$app/navigation';
+    import { base } from '$app/paths';
+    import { page } from '$app/stores';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { Modal } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, FormList, InputText } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import type { Attributes } from '../store';
+    import { options, type Option } from './store';
+
+    export let showEdit = false;
+    export let selectedAttribute: Attributes;
+
+    const databaseId = $page.params.database;
+    const collectionId = $page.params.collection;
+
+    let error: string;
+    let currentAttr: Attributes;
+
+    $: option = options.find((option) => {
+        if (selectedAttribute) {
+            if ('format' in selectedAttribute && selectedAttribute.format) {
+                return option?.format === selectedAttribute?.format;
+            } else {
+                return option?.type === selectedAttribute?.type;
+            }
+        }
+    }) as Option;
+
+    async function submit() {
+        try {
+            await option.update(databaseId, collectionId, selectedAttribute);
+            await Promise.allSettled([invalidate(Dependencies.COLLECTION)]);
```

Comment:
can you double check if this is working without `Promise.all` or `Promise.allSettled`?